libs/prelude/Prelude/Monad.idr

```diff
@@ -11,7 +11,7 @@ import IO
 
 %access public export
 
-infixl 5 >>=
+infixl 1 >>=
```

Comment:
In Haskell this operator has a lower precedence than `<*>` and `<|>`. There's good arguments for both lower and higher precedence, but having one of the lowest precedences means it will not conflict with most other operators.

It's not used much outside of `do` notation, so any impact this change has should be minor.